test/test-service-external-db.js

```diff
@@ -0,0 +1,33 @@
+var meshServer = require('../index').meshServer;
+var ServiceManager = require('../index').ServiceManager;
+var SQLite3 = require('loopback-connector-sqlite3');
+var test = require('tap').test;
+
+test('Service environment variable manipulation', function(t) {
+  var manager = new ServiceManager();
+  var DataSource = require('loopback-datasource-juggler').DataSource;
+  var db = new DataSource(SQLite3);
```

Comment:
Most of what happens in mesh-models is LB booting the app and attaching models and initializing them.